maquetado/setup/scss/navbar.scss

```diff
@@ -0,0 +1,25 @@
+$navbar-height: 90px;
+$navbar-logo-height: 30px;
+
+.navbar {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 5px 4px 10px $grey;
+  display: flex;
+  height: $navbar-height;
+  justify-content: space-between;
+  position: fixed;
+  top: 0px;
+  width: 100%;
+  z-index: 1;
+}
+
+.navbar-logo {
+  height: $navbar-logo-height;
+  margin-left: 15px;
```

Comment:
`margin: 15px` shall do this, its better not to be so specific if its not necessary